docs/content/components/form/slider/slider.mdx

```diff
@@ -1,66 +1,213 @@
 ---
 title: Slider
-caption: Component to select a value within a range.
+caption: Allows to make selections from a range of values.
+badge: updated
 ---
 
 The `<Slider>` is a form component that allows users to quickly select a value within a range. They should be used when the upper and lower bounds of the range are known and static.
+Sliders provide a visual indication of adjustable content, where the user can increase or decrease the value by moving the handle along a horizontal track.
 
-For accessibility reasons you can use the left and right arrow keys on your keyboard to select a value.
+<SectionMessage>
+  <SectionMessage.Title>Note</SectionMessage.Title>
+  <SectionMessage.Content>
+    For accessibility reasons you can use the left and right arrow keys on your
+    keyboard to select a value.
+  </SectionMessage.Content>
+</SectionMessage>
 
-## Import
+## Anatomy
 
-```tsx
-import { Slider } from '@marigold/components';
-```
+Sliders consist of a track element showing the range of available values, one or more thumbs showing the current values, an output displaying the current values textually, and an optional label.
+
+The thumbs can be dragged to allow a user to change their value. In addition, the track can be clicked to move the nearest thumb to that position.
+
+<Image
+  src="/slider/slider-anatomy.jpg"
+  alt="Anatomy of slider"
+  width={800}
+  height={550}
+  className="mx-auto block"
+/>
 
 ## Appearance
 
+<AppearanceDemo component={title} />
+
 <AppearanceTable component={title} />
 
-## Props
+## Usage
 
-<PropsTable component={title} />
+Sliders allow users to view and select a value (or range) along a track. They’re ideal for adjusting settings such as volume and brightness.
+A slider’s `step` size defines the increments at which the slider can move, allowing users to make finer or coarser adjustments based on the context—such as adjusting volume in single steps versus increments of ten. Setting an appropriate step size enhances usability, especially for precise inputs.
```

Comment:
```suggestion
A slider’s `step` size defines the increments at which the slider can move, allowing users to make finer or coarser adjustments based on the context — such as adjusting volume in single steps versus increments of ten. Setting an appropriate step size enhances usability, especially for precise inputs. The `step` default increment is 1.
```

Added whitespace to `—` (AI is forgetting that all the time ;D) and moved the default sentence to the paragraph.